src/components/Toolbar.tsx

```diff
@@ -127,6 +130,11 @@ const Toolbar: FC<ToolbarProps> = ({ customize, onSelect, selected }) => {
   const onTapUp = useCallback(
     (id: ShortcutId) => {
       deselectPressingToolbarId()
+      if (id === 'newThought') {
+        store.dispatch(showTip({ tip: TipId.NewThought }))
```

Comment:
Use `useDispatch` instead of importing the global store.

Reply:
Switched to `useDispatch`: 55fb25a8f86329f823cfd9d0316aa3cdbb6bfe17